packages/design-system-web/src/components/geov-entity/geov-entity.css

```diff
@@ -35,15 +33,15 @@ geov-entity-label {
 }
 
 .section:nth-child(even) {
-  background: var(--ion-color-tertiary);
+  background: var(--ion-color-tertiary-shade);
 }
 
 .section:nth-child(odd) {
-  background: var(--ion-color-tertiary-shade);
+  background: var(--ion-color-tertiary);
 }
 
 ion-card {
-  box-shadow: none !important
+  box-shadow: none !important;
```

Comment:
I agree, that this again, is hard to style. since the box-shdow is styled by a `:host` rule. You would need to give the ion-cards their own class, which is not easily doable at the moment. So I understand, why you used !important here.
@joschne Is there no way to give ion-elements their own custom class? This use of !important is gonna bring us in devils kitchen...

Reply:
This was not my part but I don't see a change if this is removed.
Removed

Reply:
> @joschne Is there no way to give ion-elements their own custom class? This use of !important is gonna bring us in devils kitchen...

I aggree but: 
- @perrauda was right, this can be removed here, because the styling was moved down to `geov-entity-props-by-predicate.css`
- `<geov-entity-props-by-predicate>` is using shadowDom (see code below). This ensures, that its styling is encapsulated and only affects its own children. Therefore, within shadowDom , `!important` can be used without risk IMO @flicksolutions.

```typescript
@Component({
  tag: 'geov-entity-props-by-predicate',
  styleUrl: 'geov-entity-props-by-predicate.css',
  shadow: true,
})
```




From [shadow DOM](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM)

> You can affect the nodes in the shadow DOM in exactly the same way as non-shadow nodes — for example appending children or setting attributes, styling individual nodes using element.style.foo, or adding style to the entire shadow DOM tree inside a [<style>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style) element. The difference is that none of the code inside a shadow DOM can affect anything outside it, allowing for handy encapsulation.